ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml

```diff
@@ -15,6 +15,32 @@ resources:
     requests:
       cpu: 200m
       memory: 400Mi
+tests:
+- as: base-ubi8-python-3.8-mirror
+  postsubmit: true
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: base-ubi8-python-3.8
+    env:
+      IMAGE_REPO: notebooks
+      RELEASE_VERSION: main
+    workflow: opendatahub-io-ci-image-mirror
+- as: base-ubi8-python-3.8-mirror-nightly
+  cron: 0 2 * * *
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: base-ubi8-python-3.8
+    env:
+      IMAGE_REPO: notebooks
+      IMAGE_TAG: nightly
+    workflow: opendatahub-io-ci-image-mirror
+- as: base-ubi8-python-3.8-pr-image-mirror
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: base-ubi8-python-3.8
+    env:
+      IMAGE_REPO: notebooks
```

Comment:
Can you add the `RELEASE_VERSION: base-ubi8-python-3.8` env var here too (under IMAGE_REPO)?

Additionally update the presubmit condition block in the [opendatahub-io-ci-image-mirror-commands.sh script](https://github.com/openshift/release/blob/master/ci-operator/step-registry/opendatahub-io/ci/image-mirror/opendatahub-io-ci-image-mirror-commands.sh#L28-L31) to include the RELEASE_VERSION in the image tag if set:

```shell
presubmit)
    log "INFO Building default image tag for a $JOB_TYPE job"
    IMAGE_TAG="pr-${PULL_NUMBER}"
    if [[ -n "${RELEASE_VERSION-}" ]]; then
        IMAGE_TAG="${RELEASE_VERSION}-${IMAGE_TAG}"
    fi
    ;;
```